### ISSUE TITLE
Parse Error: Function "get_plugin_version()" not found in base self.

### ISSUE THREAD

**Issue by philippludwig** [Poing-Studios/godot-admob-android#207]
### Godot version

4.1.3

### Plugin version

most recent from the asset store – 3.0.1, apparently.

### Phone information

n/a

### Issue description

After downloading the Plugin from the AssetLib, I get:

```
res://addons/admob/admob.gd:34 - Parse Error: Function "get_plugin_version()" not found in base self.
res://addons/admob/admob.gd:34 - Parse Error: Cannot infer the type of "plugin_version" variable because the value doesn't have a set type.
res://addons/admob/admob.gd:105 - Parse Error: Function "get_plugin_version()" not found in base self.
res://addons/admob/admob.gd:182 - Parse Error: Function "get_plugin_version()" not found in base self.
```

The plugin can’t be enabled.

### Steps to reproduce

1. Follow the installation instructions @ https://poing-studios.github.io/godot-admob-plugin/

**Comment by philippludwig** [comment]
Nevermind, apparently this is fixed with 3.0.2, which is sadly not yet available in the AssetLib.

**Comment by gumaciel** [comment]
Hi @philippludwig yes, the new version it's still under review by Godot team, thank you for using our plugin
